bin/editorconfig-el

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+:;#-*-Emacs-Lisp-*-
```

Comment:
Thanks! I'll add that tonight (about 6 hours later :smiley_cat:) and also merge this PR!